packages/salesforcedx-utils-vscode/webpack.config.js

```diff
@@ -0,0 +1,84 @@
+const path = require('path');
+const glob = require('glob');
+const DIST = path.resolve(__dirname);
+
+const getEntryObject = () => {
```

Comment:
Function that builds the entry object used by webpack in order to build the bundles. It adds the test file paths if you run the configuration in `development` mode.